benefits/pension_scheme.md

```diff
@@ -0,0 +1,15 @@
+# Pension Scheme
+
+We provide a pension scheme inline with the UK Government's rollout of automatic enrolment. We automatically enrol everyone in our pension scheme after their probation period (usually 3 months). You can choose to opt out as per [official guidelines](https://www.gov.uk/workplace-pensions/if-you-want-to-leave-your-workplace-pension-scheme), we give you the opportunity to specify this when we onboard you.
+
+## Pension contributions
+
+Below are the guidelines to the contributions we pay, and the minimum contribution you can pay.
+
+When | We pay | You pay | Total minimum contribution
+-----|--------|---------|----------------------------
+Now | 1% | 1% | 2%
+From April 2018 | 2% | 3% | 5%
+From April 2019 | 3% | 5% | 8%
+
+You can choose to increase your pension contribution. You can do this by speaking with a director or our finance department.
```

Comment:
Is it worth specifying how to contact the Finance department? (I think it's just finance@madetech.com, but I feel like I gleaned that magically from somewhere rather than reading it).